src/EnergyPlus/UnitarySystem.cc

```diff
@@ -7789,7 +7789,7 @@ namespace UnitarySystems {
                                                 Real64 &OnOffAirFlowRatio,     // ratio of heating PLR to cooling PLR (is this correct?)
                                                 Real64 const ZoneLoad,
                                                 Real64 &FullSensibleOutput,
-                                                bool HXUnitOn, // Flag to control HX for HXAssisted Cooling Coil
+                                                bool &HXUnitOn, // Flag to control HX for HXAssisted Cooling Coil
```

Comment:
This function switches on the HX as needed. This info needed to be passed back out and used in subsequent calls.